content/docs/tutorials/get-started/data-versioning.md

```diff
@@ -1,40 +1,45 @@
 # Data Versioning
 
-To **track** a large file or directory, place it in the <abbr>workspace</abbr>,
-and use `dvc add`:
+How cool would it be to make Git handle arbitraty large files and directories
+with the same performance as Git itself? Imagine you can do a `git clone` and
+see data files and ML model files in the workspace. Or do `git checkout` and
+switch to a different version of a 100Gb size file in a less than a second?
+
+The core part of DVC is a few commands that you can run along with Git to track
+a large file, ML model or a directory. To start tracking a file or directory,
+use `dvc add`:
 
 <details>
 
-### 👉 Expand to get an example dataset
+### ⚙️ Expand to get an example dataset
 
-Having [initialized](/doc/tutorials/get-started#initialize) a project, do this:
+Having initialized a project in the previous section, get the data file we will
+be using later like this:
 
 ```dvc
-$ dvc get --rev cats-dogs-v1 \
-          https://github.com/iterative/dataset-registry \
-          use-cases/cats-dogs -o datadir
+$ dvc get https://github.com/iterative/dataset-registry \
+          get-started/data.xml -o data/data.xml
 ```
 
-> `dvc get` can download any <abbr>data artifact</abbr> tracked in a <abbr>DVC
-> repository</abbr>. It's like `wget`, but for DVC or Git repos. In this case we
-> use a specific version (`cats-dogs-v1` tag) of our
-> [dataset registry](https://github.com/iterative/dataset-registry) repo as the
-> data source.
-
-Note that while the source data directory was called `cats-dogs/`, we are able
-to rename it locally to `datadir/`.
+We use the fancy `dvc get` command to jump ahead a bit and show how Git repo
```

Comment:
```suggestion
We use the fancy `dvc get` command to jump ahead a bit and show how DVC repos can
```

Reply:
Yeah, I want to emphasize that it is a Git repo in the first place.